src/components/messages/notification-history.tsx

```
@@ -0,0 +1,721 @@
+    trackEvent,
+    refreshNotifications
+  } = useMessageNotifications({
+    userId,
+    autoRefresh: false, // Manual refresh for history
+    enableRealTime: false // No real-time updates for history
+  });
+
+  const {
+    stats,
+    engagement,
+    loading
```

Biome (2.1.2)

[error] Shouldn't redeclare 'loading'. Consider to delete it or rename it.

'loading' is defined here:


(lint/suspicious/noRedeclare)